### PR TITLE
fix(hero): stlying changes for hero

### PR DESCRIPTION
**Pull request by kishore03109**
## Problem

As described in the ticket [here](https://linear.app/ogp/issue/ISOM-965/infocol-design-review)

### CODE REVIEW

**Comment by kishore03109** [comment]
* **#82** <a href="https://app.graphite.dev/github/pr/isomerpages/isomer-components/82?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @kishore03109 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by kishore03109** [comment]
!run chromatic

File: src/templates/next/components/complex/Hero/Hero.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ const HeroGradient = ({
 }: HeroGradientProps) => {
   return (
     <section
-      className="bg-cover bg-center bg-no-repeat h-[32rem]"
+      className="bg-cover bg-center bg-no-repeat h-auto sm:h-[22.5rem] md:h-[31.25rem]"
```
Comment (dcshzj) [inline src/templates/next/components/complex/Hero/Hero.tsx:26]: question: I think these should be max-height?
- Desktop: 500px
- Tablet: 360px
- Mobile: 240px

Comment (kishore03109) [reply src/templates/next/components/complex/Hero/Hero.tsx:26]: the ticket (here)[https://linear.app/ogp/issue/ISOM-993/hero-banner-design-review] is min h leh

File: src/templates/next/components/complex/Hero/Hero.tsx [hunk 2]
```diff
@@ -86,7 +86,7 @@ const HeroSplit = ({
   return (
     <>
       <section
-        className={`hidden md:flex h-[32rem] ${
+        className={`hidden md:flex h-auto sm:h-[22.5rem] md:h-[31.25rem] ${
```
Comment (dcshzj) [inline src/templates/next/components/complex/Hero/Hero.tsx:89]: issue: I think the rest needs to get a min-height as well?

Comment (kishore03109) [reply src/templates/next/components/complex/Hero/Hero.tsx]: ps forgot to update. 619cdbe

**Review by dcshzj — APPROVED** [review]

**Pull request merged**